tests/d2s_test.rs

```diff
@@ -51,13 +51,15 @@ fn test_ryu() {
 
 #[test]
 fn test_random() {
+    let n = if cfg!(miri) { 100 } else { 1000000 };
```

Comment:
Oh, I like this style... so far I always used `#[cfg(miri)]` for this but that means we need 4 lines to define one constant.